docs/FAQ.md

```diff
@@ -0,0 +1,117 @@
+# Frequently Asked Questions
+
+If you can't find the answer to your question here, please ask us in [Discord](https://discord.com/channels/752978124614008945).
+
+
+## General
+
+### How is the Yield Protocol monetized?
+The Yield company doesn't have a monetization plan yet. All fees in the liquidity pools go to liquidity providers.
+
+### How can I find you on uniswap?
+There is no governance token. You will not find us on uniswap. You could find the fyDai or fyDaiLP tokens on Uniswap or other exchanges, but you should get better prices for them in [Yield](app.yield.is) since we are tailored to the paticularities of trading fyDai, such as maturity.
```

Comment:
```suggestion
There is no governance token. You will not find us on Uniswap. You could find the fyDai or fyDaiLP tokens on Uniswap or other exchanges, but you should get better prices for them in [Yield](app.yield.is) since the app is tailored to the particularities of trading fyDai, such as its maturity date.
```